test/extension/extensionconfig.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: runtime.cluster.x-k8s.io/v1alpha1
```

Comment:
Can we move the two YAMLs e.g. to config/tilt? Just feels a bit out-of-place top-level